src/Pkg.jl

```diff
@@ -41,14 +41,18 @@ const RESPECT_SYSIMAGE_VERSIONS = Ref(true)
 # For globally overriding in e.g. tests
 const DEFAULT_IO = Ref{Union{IO,Nothing}}(nothing)
 
-struct UnstableIO <: IO
-    io::IO
+# See discussion in https://github.com/JuliaLang/julia/pull/52249
+const UnstableIO = IOContext{IO}
```

Comment:
This can be removed I think? Or is it used somewhere else?

Reply:
There are a few other references to `UnstableIO` in Pkg.jl. I could get rid of the alias as its uses. Is that what you mean?

Reply:
I removed references to `UnstableIO` in 9d57162